backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ProjectRepository.java

```diff
@@ -250,7 +250,7 @@ public ProjectRepository(DatabaseConnector db) {
     }
 
     public List<Project> searchByName(String name, User user) {
-        return searchByName(name, user, SummaryType.SHORT);
+        return searchByName(name, user, SummaryType.SUMMARY);
```

Comment:
I build up a testing database with around 10.000 dummy projects and did not see any significant change in performance when switching from `SummaryType.SHORT` to `SummaryType.SUMMARY`. 

But with such a large number of projects the rendering of the DataTable (sorting) becomes time-consuming...